assets/airports/kbos.json

```diff
@@ -49,77 +54,73 @@
         }
     ],
     "fixes": {
-        "_15R": ["N42.34267058224", "W70.97600868835"],
-        "_22L": ["N42.33552454364", "W71.01994630118"],
-        "_22R": ["N42.34225233814", "W71.02249160551"],
-        "_27":  ["N42.35233868916", "W71.03492600845"],
-        "_33L": ["N42.38627445008", "W71.03354932872"],
-        "_LOGAN04L":  ["N42.42409", "W70.98058"],
-        "_LOGAN04R":  ["N42.42323", "W70.97491"],
-        "_LOGAN09":   ["N42.37730732453", "W70.87770422052"],
-        "_LOGAN14":   ["N42.34112", "W70.99401"],
-        "_LOGAN15R":  ["N42.34352", "W70.97709"],
-        "_LOGAN22L":  ["N42.30321881627", "W70.91983919769"],
-        "_LOGAN22R":  ["N42.30994664062", "W70.92237384752"],
-        "_LOGAN27":   ["N42.35182", "W71.03833"],
-        "_LOGAN33L":  ["N42.37889", "W71.02394"],
-        "_LOGAN9-1":  ["N42.32617336596", "W70.89029730551"],
-        "_LOGAN9-2":  ["N42.33256591613", "W71.06025996325"],
-        "_LOGAN9-3":  ["N42.40464167947", "W71.08178558128"],
-        "_LOGAN9-4":  ["N42.42750031521", "W70.95315673987"],
-        "_LOGAN9-5":  ["N42.42836031475", "W70.95882644261"],
-        "_LOGAN9-6":  ["N42.32377334760", "W70.90722060934"],
+        "_BOS146001": ["N42.34267058224", "W70.97600868835"],
+        "_BOS226002": ["N42.33552454364", "W71.01994630118"],
+        "_BOS227002": ["N42.34225233814", "W71.02249160551"],
+        "_BOS261002": ["N42.35233868916", "W71.03492600845"],
+        "_BOS312003": ["N42.38627445008", "W71.03354932872"],
+        "_BOS036004": ["N42.42409", "W70.98058"],
+        "_BOS037004": ["N42.42323", "W70.97491"],
+        "_BOS142001": ["N42.34112", "W70.99401"],
+        "_BOS151001": ["N42.34352", "W70.97709"],
+        "_BOS273002": ["N42.35182", "W71.03833"],
+        "_BOS331002": ["N42.37889", "W71.02394"],
```

Comment:
This works. A quick note though: since we are using these to emulate straight-out climbs from the runways, it would generally be better to use the `_RWY22L01DME` if trying to achieve a distance from the departure end, or the not-yet-documented `_FIXXA150_FIXXB270` for the intersection between the 150 degree radial outbound from FIXXA and the 270 degree radial outbound from FIXXB (also could be applied as `_RWY22L_FIXXB270` for the intersection of the radial and the inbound approach course).

I suggest this for the future, simply because it will provide better context to people examining the file as to where that point is and what it is used for. We're all set here though, unless you really feel like going back through them-- I don't see the need to personally. :smiley: